docs/src/examples/components/Toolbar/Content/ToolbarExamplePortal.shorthand.tsx

```diff
@@ -0,0 +1,273 @@
+                  setBulletListActive(false)
+                  setNumberListActive(false)
+                },
+                'aria-label': 'to do list',
+              },
+            ],
+          },
+          { key: 'divider3', kind: 'divider' },
+          { key: 'outdent', icon: { name: 'outdent', outline: true } },
+          { key: 'indent', icon: { name: 'indent', outline: true } },
+          { key: 'divider4', kind: 'divider' },
+          {
+            key: 'more',
+            icon: { name: 'more', outline: true },
+            active: moreMenuOpen,
+            menu: [
+              {
+                content: 'Quote',
+                icon: 'quote',
+                onClick: () => {
+                  writeLog('... -> Quote')
+                },
+              },
+              {
+                content: 'Link',
+                icon: 'link',
+                disabled: true,
+                onClick: () => {
+                  writeLog('SHOULD NOT BE CALLED, ITEM IS DISABLED... -> Link')
+                },
+              },
+              {
+                content: 'Code snippet',
+                icon: 'code-snippet',
+                onClick: () => writeLog('... -> Code snippet'),
+              },
+            ],
+            menuOpen: moreMenuOpen,
+            onMenuOpenChange: (e, { menuOpen }) => {
+              writeLog(`setting menu to ${menuOpen ? 'open' : 'close'}`)
+              setMoreMenuOpen(menuOpen)
+            },
+          },
+        ]}
+      />
+    </>
+  )
+}
+
+const PortalWindow = ({ children, onClose }) => {
+  const externalContainer = React.useRef(null)
+  const externalWindow = React.useRef(null)
+  const [mounted, setMounted] = React.useState(false)
+  React.useEffect(() => {
+    externalWindow.current = window.open('', '', 'width=600,height=400,left=200,top=200')
+    externalContainer.current = externalWindow.current.document.createElement('div')
+    externalWindow.current.document.body.appendChild(externalContainer.current)
+    if (onClose) externalWindow.current.onbeforeunload = onClose
+    setMounted(true)
+    return () => {
+      externalWindow.current.close()
+    }
+  }, [])
+  return (
+    mounted &&
+    ReactDOM.createPortal(
+      children(externalContainer.current.ownerDocument),
+      externalContainer.current,
+    )
+  )
+}
+
+const ProviderExampleTarget = () => {
+  const [open, setOpen] = React.useState(false)
+  return (
+    <>
+      <Text>This example will be removed before merge</Text>
+      <Button onClick={() => setOpen(true)}>Open window!</Button>
+      {open && (
+        <PortalWindow onClose={() => setOpen(false)}>
+          {externalDocument => (
+            <Provider rtl theme={themes.teams} target={externalDocument}>
+              {<ToolbarExampleShorthand />}
+            </Provider>
+          )}
+        </PortalWindow>
+      )}
+
+      <ToolbarExampleShorthand />
+    </>
+  )
+}
+
+export default ProviderExampleTarget
```

Comment:
Suggested options:
- hide this example and use it in screener tests only
  - use `iframe` to render it
- remove it at all

Reply:
removed for now